include/avtp_rvf.h

```diff
@@ -51,10 +51,10 @@ extern "C" {
 #define AVTP_RVF_PIXEL_FORMAT_444 0x04
 #define AVTP_RVF_PIXEL_FORMAT_4224 0x06
 #define AVTP_RVF_PIXEL_FORMAT_4444 0x07
-#define AVTP_RVF_PIXEL_FORMAT_BAER_GRBG 0x08
-#define AVTP_RVF_PIXEL_FORMAT_BAER_RGGB 0x09
-#define AVTP_RVF_PIXEL_FORMAT_BAER_BGGR 0x0A
-#define AVTP_RVF_PIXEL_FORMAT_BAER_GBRG 0x0B
+#define AVTP_RVF_PIXEL_FORMAT_BAYER_GRBG 0x08
+#define AVTP_RVF_PIXEL_FORMAT_BAYER_RGGB 0x09
+#define AVTP_RVF_PIXEL_FORMAT_BAYER_BGGR 0x0A
+#define AVTP_RVF_PIXEL_FORMAT_BAYER_GBRG 0x0B
 #define AVTP_RVF_PIXEL_FORMAT_USER 0x0F
```

Comment:
Cool, but please squash this into the "Add RVF support" patch.

Reply:
Done.